frontend/src/components/applyForm/widgets/MultiSelectWidget.tsx

```diff
@@ -0,0 +1,216 @@
+"use client";
+
+import {
+  enumOptionsValueForIndex,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+} from "@rjsf/utils";
+
+import React, { useCallback, useMemo, useRef } from "react";
+import {
+  ComboBox,
+  ErrorMessage,
+  FormGroup,
+  type ComboBoxOption,
+  type ComboBoxRef,
+} from "@trussworks/react-uswds";
+
+import { UswdsWidgetProps } from "src/components/applyForm/types";
+import { Pill } from "src/components/Pill";
+import { DynamicFieldLabel } from "./DynamicFieldLabel";
+import { getLabelTypeFromOptions } from "./getLabelTypeFromOptions";
+
+function toArray(selectedValue: unknown): string[] {
```

Comment:
[nit] since the first case here is to turn an array into an array of strings, maybe the name should be `toStringArray`?